lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -84,6 +104,8 @@ sub test_network_interface {
     my $routed   = $args{routed}   // 0;
     my $target   = $args{target}   // script_output("dig +short openqa.suse.de");
 
+    check_guest_ip("$guest") if (is_sle('>15') || ($isolated == 1));
```

Comment:
From your description, it seems that AND logic should be used instead of OR?

Reply:
Cool! there should be AND logic.
And updated here now.